service/init.go

```diff
@@ -478,6 +487,13 @@ func (amf *AMF) BuildAndSendRegisterNFInstance() (models.NfProfile, error) {
 func (amf *AMF) UpdateNF() {
 	KeepAliveTimerMutex.Lock()
 	defer KeepAliveTimerMutex.Unlock()
+	defer func() {
+		if err := recover(); err != nil {
```

Comment:
In what scenario will we get here ?

Reply:
catching if there is any panic.. this is the sample code to recover from panic